Lesson6/gokhan/styles.css

```
@@ -0,0 +1,158 @@
+styles.css
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+}
+
+ul {
+  list-style-type: none;
+  padding-left: 0;
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 4px 28px;
+  height: 60px;
+  box-shadow: 0 0.25rem 0.75rem rgba(0, 0, 0, .05);
+}
+
+.nav-list {
+  display: flex;
+  gap: 10px;
+  font-size: 12px;
+}
+.btn1 {
+  min-width: 48px;
+  padding: 6px 12px;
+  border-radius: 2px;
```

5px should be closer to reality :) 
```suggestion
  border-radius: 5px;
```